js/models/LocalSettings.js

```diff
@@ -51,6 +51,7 @@ export default class extends Model {
       listingsGridViewType: 'grid',
       bitcoinUnit: 'BTC',
       verifiedModsProvider: 'https://search.ob1.io/verified_moderators',
```

Comment:
Is the idea that if the provider doesn't provide a seperate tor url, the standard url will be filled in for both urls?